WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Amount/SimplePaymentsAmountViewModel.swift

```diff
@@ -107,6 +108,12 @@ final class SimplePaymentsAmountViewModel: ObservableObject {
         analytics.track(event: WooAnalyticsEvent.SimplePayments.simplePaymentsFlowCanceled())
     }
 
+    /// Disable cancel button while performing the create order operation
+    ///
+    func disableCancelButton() -> Bool {
+        return loading
+    }
```

Comment:
Question: Why did you decide to make this a function instead of a computed variable?

Also if you rename this as `var disableCancel: Bool { loading }` you can use it in the hosting view controller too.